x/application/types/params.go

```diff
@@ -7,14 +7,20 @@ import (
 
 var _ paramtypes.ParamSet = (*Params)(nil)
 
+const (
+	DefaultMaxDelegatedGateways int64 = 7
+)
```

Comment:
NIT: unless we plan on adding more constants immediately after, I think we should prefer one-line `const` and `var` declarations.

```suggestion
const DefaultMaxDelegatedGateways int64 = 7
```

Reply:
NIT: I would also move the `const` above the `var`.